components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLModelHandler.java

```diff
@@ -599,6 +600,8 @@ public void run() {
                 MLModel model = modelBuilder.build();
                 
                 persistModel(id, ctxt.getModel().getName(), model);
+
+                emailTemplateParameters[1] = MLUtils.getLink(ctxt, MLConstants.MODEL_STATUS_COMPLETE);
                 EmailNotificationSender.sendModelBuildingCompleteNotification(emailNotificationEndpoint,
                         emailTemplateParameters);
             } catch (Exception e) {
```

Comment:
@cnapagoda @nuwand Fixed in pull request https://github.com/wso2/carbon-ml/pull/51. Thanks for pointing this out and for the explanation!